web/js/containers/sidebar/layers-container.js

```diff
@@ -103,8 +105,9 @@ function LayersContainer (props) {
     </DragDropContext>
   );
 
+  const maxHeight = isCompareActive && isMobile ? height - 94 : height;
```

Comment:
It looks like the compare button gets pushed off screen when there are lots of layers in the layer list:
![Screen Shot 2021-03-17 at 10 03 03 AM](https://user-images.githubusercontent.com/1480000/111480173-11059a00-8708-11eb-839d-9beba54d89d5.png)